HOW-TO-RELEASE.md

```diff
@@ -23,3 +23,30 @@ Wait for the [CI pipeline](../../actions) to release the new version. Publishing
 Open the [releases](../../releases). A draft should already be prepared.
 
 Edit the draft release to set the released version. Complete the release notes if necessary. And save it.
+
+## Troubleshooting
+
+If the `Publish Artifacts` job fails, it might be caused by an expired GPG signing key.
+In that case you'll find the following entries in the job log:
+```text
+[info] gpg: no default secret key: No secret key
+[info] gpg: signing failed: No secret key
+```
+
+To resolve the issue, you have to rotate the GPG signing key.
+Follow the instructions, found [here](https://github.com/sbt/sbt-ci-release#gpg).
+
+* for real name, use `sbt-ci-release bot`
+* for email address, use `info@commercetools.com`
+
+If the programmatic public key publishing fails for you (`gpg: keyserver send failed: Server indicated a failure`), you can publish the public key manually.
+Use the forms on https://keyserver.ubuntu.com/ and http://pgp.mit.edu:11371/.
+It's important to add the public key to both key servers!
+
+Finally, update the `PGP_PASSPHRASE` and `PGP_SECRET` secrets in the [repository settings](https://github.com/commercetools/sphere-scala-libs/settings/secrets/actions).
+
+_There is no need to save the key details anywhere else because it's only used for signing and will never be verified.
+If the credentials are lost, we can just generate a new key._
+
+If you generated the key using `gpg --gen-key`, its output included an `[expires: YYYY-MM-DD]` string.
+Please make sure to set a reminder to rotate the key before it expires.
```

Comment:
When discussing this, please keep in mind that this is a public repository. The discussion might better be done internally.